docs/src/cookbooks/cookbook.md

```diff
@@ -447,16 +447,16 @@ class Salt extends Module {
 }
 ```
 
-Elaborating the Chisel module `Salt` yields our "desire name" for `Salt` and `Coffee` in the output Verilog:
+Elaborating the Chisel module `Salt` yields our "desired names" for `Salt` and `Coffee` in the output Verilog:
+```scala mdoc:silent
+import chisel3.stage.ChiselStage
+
+(ChiselStage).emitVerilog(new Salt)
+```
+
 ```scala mdoc:passthrough
-import chisel3.stage.{ChiselStage, ChiselGeneratorAnnotation}
-import firrtl.annotations.DeletedAnnotation
-import firrtl.EmittedVerilogCircuitAnnotation
-
-(new ChiselStage)
-  .execute(Array("-X", "verilog"), Seq(ChiselGeneratorAnnotation(() => new Salt)))
-  .collectFirst{ case DeletedAnnotation(_, a: EmittedVerilogCircuitAnnotation) => a.value.value }
-  .foreach(a => println(s"""|```verilog
-                            |$a
-                            |```""".stripMargin))
+val a = (ChiselStage).emitVerilog(new Salt)
+println(s"""|```verilog
+            |$a
+            |```""".stripMargin)
 ```
```

Comment:
This is what our custom `verilog` modifier is for (https://github.com/chipsalliance/chisel3/tree/master/docs#custom-verilog-modifier)!
```scala mdoc:verilog
ChiselStage.emitVerilog(new Salt)
```

Reply:
ah, yeah, much cleaner!